### PR TITLE
Fix #676

### LINKED ISSUE #676: Sync supported framework adapters
They're outdated. This needs to be fixed.

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: site/docs/plugins/session.md [hunk 1]
```diff
@@ -558,19 +558,10 @@ bot.start();
...
-- Deta.sh Base: <https://github.com/grammyjs/storages/tree/main/packages/deta>
-- Google Firestore (Node.js-only): <https://github.com/grammyjs/storages/tree/main/packages/firestore>
-- Files: <https://github.com/grammyjs/storages/tree/main/packages/file>
-- MongoDB: <https://github.com/grammyjs/storages/tree/main/packages/mongodb>
-- Redis: <https://github.com/grammyjs/storages/tree/main/packages/redis>
-- PostgreSQL: <https://github.com/grammyjs/storages/tree/main/packages/psql>
-- TypeORM (Node.js-only): <https://github.com/grammyjs/storages/tree/main/packages/typeorm>
-- DenoDB (Deno-only): <https://github.com/grammyjs/storages/tree/main/packages/denodb>
-- Prisma (Node.js-only): <https://github.com/grammyjs/storages/tree/main/packages/prisma>
+Visit [here](https://github.com/grammyjs/storages/tree/main/packages#grammy-storages) to see a list of currently supported adapters and get guide on using them.
```
Comment (KnorpelSenf) [inline site/docs/plugins/session.md:564]:
```suggestion
Visit [here](https://github.com/grammyjs/storages/tree/main/packages#grammy-storages) to see a list of currently supported adapters and get guidance on using them.
```

Comment (roj1512) [reply site/docs/plugins/session.md]: Gut catch

**Review by KnorpelSenf — APPROVED** [review]
TYSM!

**Review by habemuscode — APPROVED** [review]

File: site/docs/id/plugins/session.md [hunk 2]
```diff
@@ -559,19 +559,10 @@ bot.start();
...
-- MongoDB: <https://github.com/grammyjs/storages/tree/main/packages/mongodb>
-- Redis: <https://github.com/grammyjs/storages/tree/main/packages/redis>
-- PostgreSQL: <https://github.com/grammyjs/storages/tree/main/packages/psql>
-- TypeORM (hanya untuk Node.js): <https://github.com/grammyjs/storages/tree/main/packages/typeorm>
-- DenoDB (hanya untuk Deno): <https://github.com/grammyjs/storages/tree/main/packages/denodb>
-- Prisma (hanya untuk Node.js): <https://github.com/grammyjs/storages/tree/main/packages/prisma>
+Kami memiliki daftar storage adapter resmi yang bisa kamu gunakan untuk menyimpan data session di berbagai tempat.
+Beberapa di antaranya mengharuskan kamu untuk mendaftar di sebuah penyedia layanan hosting, ataupun meng-hosting storage-mu sendiri.
+
+Kunjugi repository [berikut](https://github.com/grammyjs/storages/tree/main/packages#grammy-storages) untuk melihat adapter apa saja yang didukung serta mempelajari cara penggunaannya.
```
Comment (fadzikri) [inline site/docs/id/plugins/session.md:565]:
```suggestion
Kunjungi repository [berikut](https://github.com/grammyjs/storages/tree/main/packages#grammy-storages) untuk melihat adapter apa saja yang didukung serta mempelajari cara penggunaannya.
```

Comment (fadzikri) [reply site/docs/id/plugins/session.md:565]: just it.

**Review by fadzikri — CHANGES_REQUESTED** [review]

**Review by fadzikri — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
@roj1512 something fails with the translation guard (which for some reason is called test now)

**Pull request merged**